base/cvd/cuttlefish/host/commands/cvd/cli/commands/reset.cpp

```diff
@@ -158,6 +158,11 @@ class CvdResetCommandHandler : public CvdCommandHandler {
     }
     CF_EXPECT(KillAllCuttlefishInstances(
         /* clear_instance_dirs*/ options.clean_runtime_dir));
+
+    if (DirectoryExists("/tmp/acloud_cvd_temp/")) {
```

Comment:
As there are already [3 non-test occurrences](https://github.com/search?q=repo%3Agoogle%2Fandroid-cuttlefish%20acloud_cvd_temp&type=code) of this string, maybe worth making a shared constant in a new file somewhere?